demos/gaze_estimation_demo/cpp_gapi/main.cpp

```diff
@@ -164,6 +163,8 @@ int main(int argc, char *argv[]) {
             fileNameNoExt(FLAGS_m_fd) + ".bin",  // path to weights
             FLAGS_d_fd,                          // device specifier
         };
+        slog::info << "Network " << FLAGS_m_fd << " is loaded to " << FLAGS_d_fd << " device." << slog::endl;
```

Comment:
When demo uses more than one model, it could be useful to print model type here, e.g.:
```
[ INFO ] The face detection model|network|net face.xml is loaded to CPU
```
and some errors would be more obvious:
```
[ INFO ] The face detection model|network|net age-gender-1337.xml is loaded to CPU
```

Reply:
Added.